gateway/event_handler_webhooks.go

```diff
@@ -63,15 +66,22 @@ func (w *WebHookHandler) createConfigObject(handlerConf interface{}) (config.Web
 
 // Init enables the init of event handler instances when they are created on ApiSpec creation
 func (w *WebHookHandler) Init(handlerConf interface{}) error {
-	var err error
-	w.conf, err = w.createConfigObject(handlerConf)
+	whConf, err := w.createConfigObject(handlerConf)
```

Comment:
This should be `err := w.conf.Decode(handlerConf)` now?

Reply:
`createConfigObject` is not changed/deleted. This change is required to respect the newly added `disabled` flag.

Reply:
you can still do that, but delete createConfigObject(), w.conf.Scan has the same marshall/unmarshall logic and would populate w.conf according to the input